core/src/main/resources/lib/layout/search-bar_pl.properties

```diff
@@ -0,0 +1,23 @@
+# The MIT License
+#
+# Copyright (c) 2023, Sun Microsystems, Inc., Damian Szczepanik
```

Comment:
Ditto:

```suggestion
# Copyright (c) 2023, Damian Szczepanik
```

Reply:
Unresolving. This still seems to read "Sun Microsystems" in the latest version of this PR.